packages/ag-charts-community/src/chart/toolbar/toolbarStyles.css

```diff
@@ -53,6 +53,8 @@
 .ag-charts-toolbar--floating {
     padding: 0;
     width: auto;
+    /* Needs to beat .ag-crosshair-label */
+    z-index: 99999;
```

Comment:
css variable in `domStyles.css`? like `--ag-charts-layer-tooltip`

Reply:
I’m hoping a temp fix because I want to go through and fix all our z indices